hbase-resource-bundle/src/main/resources/supplemental-models.xml

```diff
@@ -2179,4 +2179,46 @@ Copyright (c) 2007-2017 The JRuby project
       </licenses>
     </project>
   </supplement>
+  <supplement>
+    <project>
+      <groupId>jakarta.validation</groupId>
```

Comment:
are these coming in from the new shaded jetty? can we shade them there rather then keep them as transitive dependencies?

Reply:
The classes are located under javax, so I assume they are just repackaging of some javax classes, so I exclude them in the shaded artifacts.